src/formatDate.js

```diff
@@ -50,7 +50,56 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+  let currentYear = '';
+  let currentMonth = '';
+  let currentDay = '';
+  const oldDateArr = date.split(oldSeparator);
+  const newDateArr = [];
+
+  fromFormat.map((el, index) => {
```

Comment:
`map` method should always return value for each iteration. here would be better to use `forEach`